qemu/tests/cfg/ioeventfd.cfg

```diff
@@ -1,29 +1,24 @@
 - ioeventfd: install setup image_copy unattended_install.cdrom
-    type = virt_subtest_combine
+    type = ioeventfd
     only virtio_blk virtio_scsi
     kill_vm_on_error = yes
     login_timeout = 240
-    subtests = dd_test boot autotest_control shutdown
-    reboot_method = shell
-    reboot_count = 1
-    test_timeout_reboot = 4200
-    test_control_file = iozone.control
-    shutdown_method = shell
-    subtest_type_dd_test = guests
-    cmd_dd_test = "dd if=/dev/zero of=/home/test.img bs=1M count=1024 oflag=direct"
-    Windows:
-        subtests = dd_test boot iozone_windows shutdown
-        cmd_dd_test = "D:\coreutils\DummyCMD.exe C:\dd_test 1073741824 1 && del C:\dd_test"
-        iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f c:\testfile"
-        iozone_timeout = 7200
+    start_vm = no
+    orig_ioeventfd = "ioeventfd=off"
+    new_ioeventfd = "ioeventfd=on"
     variants:
-        - ioeventfd_on:
-            virtio_blk:
-                blk_extra_params_image1 = "ioeventfd=on"
-            virtio_scsi:
-                bus_extra_params_image1 = "ioeventfd=on"
-        - ioeventfd_off:
-            virtio_blk:
-                blk_extra_params_image1 = "ioeventfd=off"
-            virtio_scsi:
-                bus_extra_params_image1 = "ioeventfd=off"
+        - under_stress:
+            io_stress = yes
+            reboot = yes
+            iozone_timeout = 7200
+            op_timeout = 1200
+            Linux:
+                op_cmd = 'dd if=/dev/zero of=/home/test_ioeventfd bs=1M count=10240 oflag=direct && rm -rf /home/test_ioeventfd'
```

Comment:
suggest replace all 'op' with 'dd', include 2 type io tests for this case: iozone test and dd test, op should mean operation, many types tests should be included, but here only 1 dd test.

Reply:
ok. will update.